naptime/src/test/pegasus/org/coursera/naptime/ari/graphql/models/FakeModel.courier

```diff
@@ -0,0 +1,9 @@
+namespace org.coursera.naptime.ari.graphql.models
+
+/**
+ * A fictional model used for testing purposes
+ */
+record FakeModel {
```

Comment:
Looks like this model is only used for the `unionField` capability, to be explicit can we name it something like `ModelWithTypeRefInUnion`?